configuration/chairman/byron-shelley/configuration.yaml

```diff
@@ -260,30 +262,31 @@ options:
   # go to the default backend, only to the explicitly listed backends.
   mapBackends:
     cardano.node.BlockFetchDecision.peers:
-      - TraceForwarderBK
+      # - TraceForwarderBK
```

Comment:
you can keep it here; it won't hurt. Once the backend is activated in "setupBackends", then it will become active. If the backend is not loaded, then this configuration does not have an effect.

Reply:
the benefit for the user: only one change will turn on the "TraceForwarderBK" and thus the routing to _RTview_